src/backend/utils/misc/guc.c

```diff
@@ -3176,6 +3177,17 @@ static struct config_bool ConfigureNamesBool[] =
 	},
 
 	{
+	    {"allow_file_count_bucket_num_mismatch", PGC_POSTMASTER, CLIENT_CONN_LOCALE,
+	          gettext_noop("allow hash table to be treated as random when file count and"
```

Comment:
allow file count and bucket number mismatch by viewing hash distributed table as randomly distributed table